src/lib/numericalInegrator/ERK.h

```diff
@@ -0,0 +1,98 @@
+#ifndef _ERK_H_
+#define _ERK_H_
+#include "Ode.h"
+
+class RK4 {
+	 public:
+    RK4 (

```

Comment:
Remove the trailing underscore from the constructor arguments to match with the style guide we're following. Same for RK8.